Gemfile

```diff
@@ -1,9 +1,20 @@
 # foreman plugins import this file therefore __FILE__ cannot be used
 FOREMAN_GEMFILE = __FILE__ unless defined? FOREMAN_GEMFILE
 
+require_relative 'config/boot_settings'
+
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.8'
+case SETTINGS[:rails]
+when '4.2'
+  gem 'rails', '4.2.8'
+when '5.0'
+  gem 'rails', '5.0.3'
```

Comment:
@dLobatog I assume we could use a newer SCL once its avail on CentOS/RHEL.
@domcleal are you planning to maintain a rails 5 scl? afaiu there is a ruby 2.4 scl too (which we might change to too?)